utils/changelog.js

```diff
@@ -48,8 +72,23 @@ async function main() {
     pkgs.add('vercel');
   }
 
-  const pub = Array.from(pkgs).join(',');
+  return pkgs;
+}
+
+// git log --pretty=format:"- %s [%an]" `git show-ref -s 'now@16.7.3'`...HEAD | grep -v '\- Publish '
```

Comment:
This comment is no longer accurate, right?

Reply:
https://github.com/vercel/vercel/pull/7512